scripts/build.py

```diff
@@ -2195,8 +2209,15 @@ def _form_env(self):
         return env
 
 if __name__ == '__main__':
-    boto_path = os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(__file__)), '../packages/boto-2.28.0-py2.7.whl'))
+    urllib_path = os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(__file__)), '../packages/urllib3-2.2.1-py3-none-any.whl'))
+    botocore_path = os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(__file__)), '../packages/botocore-1.34.69-py3-none-any.whl'))
+    boto_path = os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(__file__)), '../packages/boto3-1.34.69-py3-none-any.whl'))
+    data_path = os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(__file__)), '../packages/boto3_data'))
+    sys.path.insert(0, urllib_path)
+    sys.path.insert(0, botocore_path)
     sys.path.insert(0, boto_path)
+    os.environ['AWS_DATA_PATH'] = data_path
```

Comment:
Not sure if we should fix it now (old issue), but it feels strange that we install the .whl's (when running `install_ext`), yet still do this manual touch up.

Ideally, we should only need s3, when we run those commands.

Reply:
Removed.